contracts/cw20-atomic-swap/src/contract.rs

```diff
@@ -480,11 +480,10 @@ mod tests {
             preimage: hex::encode(b"This is 32 bytes, but incorrect."),
         };
         let res = execute(deps.as_mut(), mock_env(), info.clone(), release);
-        match res {
-            Ok(_) => panic!("expected error"),
-            Err(ContractError::InvalidPreimage {}) => {}
-            Err(e) => panic!("unexpected error: {:?}", e),
-        }
+        assert!(matches!(
+            res.unwrap_err(),
+            ContractError::InvalidPreimage {}
```

Comment:
Hmmm... even the shorter error messages break on multiple lines... maybe try the `.unwrap_err()` on the line above (482)